iaso/tests/api/org_unit_change_requests/test_org_unit_change_requests.py

```diff
@@ -330,8 +330,8 @@ def test_export_to_csv(self):
         """
         It tests the CSV export for the org change requests list.
         """
-        m.OrgUnitChangeRequest.objects.create(org_unit=self.org_unit, new_name="Foo")
-        change_request = m.OrgUnitChangeRequest.objects.create(org_unit=self.org_unit, new_name="Bar")
+        change_request = m.OrgUnitChangeRequest.objects.create(org_unit=self.org_unit, new_name="Foo")
+        m.OrgUnitChangeRequest.objects.create(org_unit=self.org_unit, new_name="Bar")
```

Comment:
@kemar I forgot the ask the first time round: The order of the result from this test changed, this change fixed it. Do you think this is okay or did I break something with this?

Reply:
@bramj that's really strange because the query seems to `ORDER BY "iaso_orgunit"."name" ASC`…

Anyway it's unlikely you broke something. My guess is the test has a bit of flakiness.